bin/organelle_contamination_recommendation.py

```diff
@@ -10,21 +10,13 @@
 
 def main():
     parser = argparse.ArgumentParser(description="Generate text for ENA submission")
```

Comment:
`Generate text for ENA submission` is a remnant from James Torrance's old code, I think this should be changed to something like `Create an organellar contamination report text file based on the organellar contamination BED file`

Reply:
Done